proposals/lip-0045.md

```diff
@@ -265,6 +265,7 @@ EMPTY_CCM = {
 | `ChainID` | bytes | Must be of length `CHAIN_ID_LENGTH`. | ID of a chain. |
 | `TokenID` | bytes | Must be of length `TOKEN_ID_LENGTH`. | Used for token identifiers. |
 | `CCM` | object | Must follow the [`crossChainMessageSchema`][lip-0049#ccmschema] schema. | The type of cross-chain messages. |
+| `CCU` | object | Must follow the `transactionSchema` schema, except that the `params` property satisfies `crossChainUpdateTransactionParams` instead of being of type `byte`. | The type of a cross-chain update transaction with deserialized `params` property. |
```

Comment:
```suggestion
| `CCU` | object | Must follow the `transactionSchema` schema, except that the `params` property satisfies `crossChainUpdateTransactionParams` instead of being of type `bytes`. | The type of a cross-chain update transaction with deserialized `params` property. |
```